package/gluon-config-mode-contact-info/luasrc/lib/gluon/config-mode/wizard/0500-contact-info.lua

```diff
@@ -5,16 +5,9 @@ return function(form, uci)
 
 	local owner = uci:get_first("gluon-node-info", "owner")
 
-	local s = form:section(Section, nil, pkg_i18n.translate(
-		'Please provide your contact information here to allow others to contact '
-		.. 'you. Note that this information will be visible <em>publicly</em> on '
-		.. 'the internet together with your node\'s coordinates. This means it can '
-		.. 'be downloaded and processed by anyone. This information is '
-		.. 'not required to operate a node. If you chose to enter data, it will be '
-		.. 'stored on this node and can be deleted by yourself at any time.'
-	))
+	local s = form:section(Section, nil, translate("gluon-config-mode:contact-help"))
 
-	local o = s:option(Value, "contact", pkg_i18n.translate("Contact info"), pkg_i18n.translate("e.g. E-mail or phone number"))
+	local o = s:option(Value, "contact", pkg_i18n.translate("Contact info"), translate("gluon-config-mode:contact-note"))
```

Comment:
No unqualified *translate* function exists anymore in the Gluon master, so this can't work. A quick grep for "site_i18n" should show you how it is done now.

I would prefer not to remove the old text, but only replace it if it is set in the site i18n, defaulting to the old text if unset. You can use the *_translate* function of the i18n module that returns *nil* if a translation is unset; this is preferable for all site i18n texts anyways, so the *gluon-config-mode:contact-note* is never actually shown.

Reply:
Oh sorry, I guess I shouldn't just have ported this from the 2017.1.x branch.

I have found calls to `site_i18n._translate('gluon-config-mode:reboot')`, so I guess that's what I am supposed to use now. But where does the default text (in case nothing is set) go then?

Reply:
The default text can stay as is, so the call would become something like
```lua
site_i18n._translate('gluon-config-mode:reboot') or pkg_i18n.translate('... the current text ...')
```

Don't forget to initialize site_i18n, as it is not used in *0500-contact-info.lua* so far.